frame/identity/src/tests.rs

```diff
@@ -339,31 +336,31 @@ fn killing_slashing_should_work() {
 #[test]
 fn setting_subaccounts_should_work() {
 	new_test_ext().execute_with(|| {
-		let mut subs = vec![(20, Data::Raw(vec![40; 1]))];
+		let mut subs = vec![(20, Data::Raw(vec![40; 1].try_into().unwrap()))];
```

Comment:
I'm seeing a lot of changes with this pattern. Maybe a test-scoped function like

```rust
fn one_raw_data<S>(n: u8) -> Data {
  Data::Raw(vec![n; 1].try_into().unwrap())
}
```

would be convenient?

Reply:
the other alternative is to turn the other side into `vec`, or by creating an implementation of `PartialEq` of boundedvec and vec?

Reply:
Implementing `PartialEq` actually makes a lot more sense, I'll do that instead.

Reply:
Ok, this actually doesn't work, because the problem isn't so much that `PartialEq` wasn't implemented between `Vec` and `BoundedVec`, it's that `Data::Raw` accepts a `BoundedVec` instead of a `Vec` as its inner struct.